tests/utils/misc/misc.go

```diff
@@ -38,7 +38,7 @@ func WaitForExpectedState(fn getVMPowerStatus, vmName, expectedState string) boo
 
 	log.Printf("Confirming [%s] power status for vm [%s]\n", expectedState, vmName)
 	for attempt := 0; attempt < maxAttempt; attempt++ {
-		SleepForSec(3)
+		SleepForSec(5)
```

Comment:
Sure @govint  I will move out as const ... but will keep the same value instead of increasing to reduce the time .. we are expecting this operation to be finished sooner. 

Please let me know if there is any advantage for increasing exponentially.